### PR TITLE
Change actions

### PR DESCRIPTION
**Pull request by lucasjoao**
vercel was dropped because of teams plan billing price, so I changed build action to check site builds.

also I added a new action to deploy bold's site

### CODE REVIEW

File: .github/workflows/publish-site.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      tag:
+        description: 'Target tag'
+        required: true
+
+jobs:
+  build:
+    name: Publish site
+    runs-on: self-hosted
+
+    steps:
```
Comment (calistro) [inline .github/workflows/publish-site.yml:15]: Isn't missing the `checkout step`?

**Review by calistro — DISMISSED** [review]
:top: 
I only have one question! But congrats for the initiative.

File: .github/workflows/publish-site.yml (hunk 1 above)
Comment (lucasjoao) [reply .github/workflows/publish-site.yml:15]: yeah, nice catch

**Review by calistro — APPROVED** [review]
:100:

**Pull request merged**